src/pages/FundManage/FundManage.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState } from 'react';
+import { Outlet } from 'react-router-dom';
+import FundBanner from '../../components/Fund/FundBanner';
+import openSeaLogo from "../../img/fundLogos/openSeaLogo.png";
+import netFineLogo from "../../img/fundLogos/netFineLogo.png";
+import funConLogo from "../../img/fundLogos/funConLogo.png";
+import btcImg from "../../img/icons/BTCIcon.svg";
+import ethImg from "../../img/icons/ETHIcon.svg";
+import curveImg from "../../img/icons/curveIcon.svg";
+import dopexImg from "../../img/icons/dopexIcon.svg";
+
+interface fundType {
+    id: number;
+    name: string;
+    logo: string;
+    investors: number;
+    expiresIn: number;
+    portfolioValue: number;
+    startingValue: number;
+    dataUpdated: string;
+    newlyAddedMoney: number;
+    upPercentage: number;
+    wallet: number;
+    walletAddress: string;
+    assetBalances: object;
+    protocolBalances: object;
+}
+
+const funds = [
+    {
+        id: 0,
+        name: "OpenSea Fund",
+        logo: openSeaLogo,
+        investors: 107,
+        expiresIn: 751,
+        portfolioValue: 234567,
+        startingValue: 400,
+        dataUpdated: '1 min',
+        newlyAddedMoney: 2560.78,
+        upPercentage: 14.67,
+        wallet: 8323,
+        walletAddress: '8BMMk4gdD263q7QJt3VLWnG2x1mt9HV56b4vX774n4Sc',
+        assetBalances: [
+            {
+                name: "Bitcoin",
+                shortName: "wBTC",
+                balance: 30,
+                dollarValue: 584991,
+                down: 2600,
+                percentage: 24.67,
+                img: btcImg, 
+            },
+            {
+                name: "Ethereum",
+                shortName: "ETH",
+                balance: 40,
+                dollarValue: 811,
+                down: 500,
+                percentage: 13.97,
+                img: ethImg,
+            },
+        ],
+        protocolBalances: [
+            {
+                name: "Curve",
+                dollarValue: 564,
+                percentage: 23.6,
+                img: curveImg,
+            },
+            {
+                name: "Dopex",
+                dollarValue: 2348,
+                percentage: 5.8,
+                img: dopexImg,
+            },
+        ],
+    } as fundType,
```

Comment:
avoid using `as`. if you do that, typescript trusts you on the type and we dont get the benefit of type checking. 
Instead do `const funds: Fund[]` then it will check if each item is of type fund.

Reply:
Got it, didn't know that. Thanks.